README.md

```diff
@@ -24,13 +24,24 @@ A data-driven `UICollectionView` framework for building fast and flexible lists.
 
 ## Installation
 
+### Cocoapods
 The preferred installation method for `IGListKit` is with [CocoaPods](http://cocoapods.org). Simply add the following to your Podfile:
 
 ```ruby
 # Latest release of IGListKit
 pod 'IGListKit'
 ```
 
+### Carthage
+[Carthage](https://github.com/Carthage/Carthage) is a decentralized dependency manager that builds your dependencies and provides you with binary frameworks.
```

Comment:
let's remove this line, and move the Carthage link to the "Carthage" text below 😄